apiserver/params/params.go

```diff
@@ -314,6 +314,18 @@ type OperatorProvisioningInfo struct {
 	CharmStorage KubernetesFilesystemParams `json:"charm-storage"`
 }
 
+// IssueOperatorCertificate
+type IssueOperatorCertificate struct {
+	ApplicationName string `json:"application"`
+}
+
+// IssueOperatorCertificateResult
+type IssueOperatorCertificateResult struct {
```

Comment:
The pattern is that we include in a result struct the payload and an *Error.
The api server calls are bulk (pass in a slice of tags or whatever) and we return a slice of result structs. We loop over the args and set each result struct accordingly.